interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php

```diff
@@ -486,6 +487,54 @@ public function getInformant($pid, $encounter)
         return $informant;
     }
 
+    public function getDocumentParticipants($pid, $encounter) {
+
+        $participants = '<document_participants>';
+        $participants .= $this->getDocumentReferralParticipant($pid, $encounter);
+        $participants .= $this->getOfficeContact($pid, $encounter);
+        $participants .= '</document_participants>';
+        return $participants;
+    }
+
+    public function getDocumentReferralParticipant($pid, $encounter) {
+        $participant = '';
+        $records = $this->getReferralRecords($pid);
+        if (empty($records[0]['refer_from']) || !is_numeric($records[0]['refer_from'])) {
+            return $participant;
+        }
+        $referral = $records[0];
+        $details = $this->getDetails(intval($referral['refer_from']));
+        if (empty($details)) {
+            return '';
+        } else {
+            $organization_uuid = UuidRegistry::uuidToString($details['facility_uuid']);
+        }
+
+        // TODO: get this working later
+//        $referralDate = DateFormatterUtils::dateStringToDateTime($referral['refer_date']);
+        $referralDate = date('YmdHisO');
```

Comment:
Could be date('Y-m-d H:i:sO') then it will get sorted in moment if fDate() isn't used. Seems we have three diff ISO formats where moment bulks at the YmdHisO format. Still fDate will format correctly.

Reply:
sounds good, I'll give it a try.